ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -427,60 +772,89 @@ def needDenormalized(self):
     # overload as needed in inheritors
     return True
 
-  ###############
-  # Run Methods #
-  ###############
-
-  def _useRealization(self, info, rlz):
-    """
-      Used to feedback the collected runs into actionable items within the sampler.
-      This is called by localFinalizeActualSampling, and hence should contain the main skeleton.
-      @ In, info, dict, identifying information about the realization
-      @ In, rlz, xr.Dataset, new batched realizations
-      @ Out, None
-    """
-    # The whole skeleton should be here, this should be calling all classes and _private methods.
+  def singleObjectiveConstraintHandling(self, info, rlz):
     traj = info['traj']
```

Comment:
Missing dockstring. @JunyungKim, this gets caught in the pylint test.
<img width="1103" alt="Screenshot 2024-06-08 at 10 36 30 AM" src="https://github.com/idaholab/raven/assets/52417034/ff9c9726-051d-492d-af65-b9a769cefd43">